src/pal/src/cruntime/printf.cpp

```diff
@@ -45,6 +45,9 @@ static int SscanfFloatCheckExponent(LPCSTR buff, LPCSTR floatFmt,
                                       void * voidPtr, int * pn);
 #endif // SSCANF_CANNOT_HANDLE_MISSING_EXPONENT
 
+using namespace CorUnix;
+int CoreVsnprintf(CPalThread *pthrCurrent, LPSTR Buffer, size_t Count, LPCSTR Format, va_list ap);
```

Comment:
Actually I can see now that we have InternalVsnprintf exposed in the printfcpp.hpp and it is just a wrapper for the CoreVsnprintf to make it public, so it would be cleaner to use that instead of adding the function prototype.